src/assets/toolkit/scripts/lib/component/fact-slider.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+export class FactSlider {
+  constructor (element, {
+    facts = element.querySelectorAll('.FunFact'),
+    factNavNext = element.querySelector('.js-FunFact-next'),
+    factNavPrev = element.querySelector('.js-FunFact-prev'),
+    factNavCurrent = element.querySelector('.js-FunFact-current'),
+    factNavTotal = element.querySelector('.js-FunFact-total'),
+    className = 'is-visible'
+  } = {}) {
+
+    facts = Array.from(facts);
+    let numFacts = facts.length;
+    let counter = 0;
```

Comment:
So, this...

```js
Object.assign(this, {
  element,
  facts,
  numFacts,
  factNavNext,
  factNavPrev,
  factNavCurrent,
  factNavTotal,
  className,
  counter
});
```

Is just a lazy way of accomplishing this...

```js
this.element = element;
this.facts = facts;
this.numFacts = numFacts;
this.factNavNext = factNavNext;
this.factNavPrev = factNavPrev;
this.factNavCurrent = factNavCurrent;
this.factNavTotal = factNavTotal;
this.className = className;
this.counter = counter;
```

I'm checking out the branch to see if I can tell why it would _break_ moving away from `let`. That's bizarre.